java/Oleksii-Riabov/src/com/zoolatech/lecture1/tasks/_2/Task2.java

```diff
@@ -0,0 +1,50 @@
+package com.zoolatech.lecture1.tasks._2;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a distance in meters and time in seconds
+ * and outputs a speed in meters/second and kilometers/hour. The result
+ * should be a floating point number.
+ */
+
+public class Task2 {
+
+    public static void main(String[] args) {
+
+        double distanceInMeters;
```

Comment:
try to define variables `distanceInMeters` and `timeInSeconds` as close to their usage as possible

Reply:
Already define variables in all tasks as close to their usage as possible.